doc/design/subscription-config.md

```diff
@@ -0,0 +1,50 @@
+# Subscription Config
+
+## Configuring Operators deployed by OLM
+
+It is possible to configure how OLM deploys an operator via the `config` field in the [subscription](../../pkg/api/apis/operators/subscription_types.go) object.
+
+Currently, the `SubscriptionConfig` object allows users to configure the following features on Deployments created by OLM:
```

Comment:
nit: maybe just `the Subscription Config ` so that it doesn't seem like SubscriptionConfig is another `Kind`?